rules/windows/process_creation/proc_creation_win_rar_susp_greedy.yml

```
@@ -0,0 +1,40 @@
+title: RAR Greedy Compression 
+id: afe52666-401e-4a02-b4ff-5d128990b8cb
+status: experimental
+description: Detects RAR usage that creates an archive from a suspicious folder, either a system folder or one of the folders often used by attackers for atging purposes
+references:
+  - https://decoded.avast.io/martinchlumecky/png-steganography
+author: X__Junior, Florian Roth
+date: 2022/12/15
+tags:
+  - attack.execution
+  - attack.t1059
+logsource:
+  product: windows
+  category: process_creation
+detection:
+  # example : rar.exe a -m5 -r -y -ta20210204000000 -hp1qazxcde32ws -v2560k Asia1Dpt-PC-c.rar c:\\*.doc c:\\*.docx c:\\*.xls c:\\*.xlsx c:\\*.pdf c:\\*.ppt c:\\*.pptx c:\\*.jpg c:\\*.txt >nul
+  selection_opt1:
+    - Image|endswith: '\rar.exe'
+    - Description: 'Command line RAR'
+  selection_opt2:
+    CommandLine|contains:
+      - '.exe a '
+      - ' a -m'
+  selection_cmdline:
+    CommandLine|contains|all:
+      - ' -hp' # password
+      - ' -r ' # recursive
+  selection_folders:
+    CommandLine|contains:
+      - ' C:\\*.'
```

- We said when we escape one thing in a string we escape everything for consistency
- I added the other option since you can do both of these `C:\*.` or `C:\\*.`
```suggestion
      - ' C:\\\*.'
      - ' C:\\\\\*.'
```